istioctl/pkg/writer/envoy/configdump/listener.go

```diff
@@ -313,18 +335,18 @@ func (c *ConfigWriter) PrintListenerSummary(filter ListenerFilter) error {
 			for _, match := range matches {
 				if includeConfigType {
 					l.Name = fmt.Sprintf("listener/%s", l.Name)
-					fmt.Fprintf(w, "%v\t%v\t%v\t%v\t%v\n", l.Name, address, port, match.match, match.destination)
+					fmt.Fprintf(w, "%v\t%v\t%v\t%v\t%v\n", l.Name, strings.Join(addresses, ","), port, match.match, match.destination)
```

Comment:
The output LGTM. One question: do we need to sort the addresses here in some order? Are ipv6 addresses always in additionalAddresses?

Reply:
I think it's unnecessary to sort the order because the order of the IP addresses may be dominated by many factors, such as the k8s cluster's setting and how the application is deployed.